src/lua.c

```diff
@@ -44,11 +44,6 @@
 
```

Comment:
The original PR also removed the `DONT_POISON_SPRINTF_YET` definition. Should we do that as well?

![image](https://user-images.githubusercontent.com/5287526/66798646-acc02480-eedc-11e9-957b-96b2cf22810b.png)